index.html

```diff
@@ -2128,6 +2256,213 @@ <h4>Bidirectional text</h4>
+        <p>Since it is a requirement of DAPT that <a>DAPT Documents</a> include a
+        <code>ttp:contentProfiles</code> attribute on the root element,
+        and that the attribute includes a DAPT <a>content profile</a> designator,
+        as specified at <a href="#ttp-contentprofiles"></a>,
+        it follows that any TTML document that does not include such an attribute,
+        or does not include such a profile designator,
+        can be considered not to be a <a>DAPT Document</a>;
+        therefore a processor requiring strict adherence to DAPT could stop processing
+        such a document.</p>
+      </section>
+      <section>
+        <h3>Document conformance claims on output</h3>
+        <p>If a processor takes as its input a <a>DAPT document</a> that claims conformance to
+          one or more content profiles which that processor does not support,
+          allows the document to be modified, and then outputs a document based on that
+          modified source, <a href="#ttp-contentprofiles"></a> prohibits it from including in <code>ttp:contentProfiles</code>
+          the values of those content profiles which it does not support.
+        </p>
+        <p>Conversely, a processor that takes as its input a <a>DAPT document</a> that
+          contains vocabulary relating to features that it does support,
+          but where support for those features is excluded from the content profiles
+          to which the document claims conformance, SHOULD NOT implement
+          those features in the context of the document, and MAY take
+          additional validation steps to check the semantics of the document.</p>
+      </section>
+      <section>
+        <h3>Handling <code>&lt;div&gt;</code> and  <code>&lt;p&gt;</code> elements</h3>
+        <p>[[TTML2]] allows <code>&lt;div&gt;</code> elements to contain any combination of
+          <code>&lt;div&gt;</code> elements and <code>&lt;p&gt;</code> elements.
+          The DAPT data model only describes the case where the <code>&lt;body&gt;</code> element
+          contains a <code>&lt;div&gt;</code> element that contains zero or more <code>&lt;p&gt;</code> elements.
+          In addition, attributes not corresponding to properties in the DAPT data model are permitted.</p>
+        <p>This gives rise to possibilities such as:</p>
+        <ul>
+          <li><code>&lt;div&gt;</code> elements that contain children that are not <a>Text</a> objects,
+            for example other <code>&lt;div&gt;</code> elements or <code>&lt;metadata&gt;</code> elements;</li>
+          <li><code>&lt;div&gt;</code> elements that do not have attributes required for <a>Script Events</a>, like <code>xml:id</code>;</li>
+          <li><code>&lt;div&gt;</code> elements that have attributes absent from the DAPT data model for <a>Script Events</a>,
+            like <code>begin</code> or <code>end</code>;</li>
```

Comment:
It is permitted for a Script Event to not have begin or end. I don't follow this example.

Reply:
Good catch.

Reply:
Changed to use `xml:space` in f06bfb2.